### PR TITLE
Mixed

### PR DESCRIPTION
**Pull request by waynebruce0x**

### CODE REVIEW

**Comment by 0xngmi** [comment]
i checked rvirkitty yesterday and it seems like their contract deposits the pool2 tokens into hucleberry's masterchef, so counting the pool2 tokens on their masterchef would miss some tokens (would also be great to split these into a pool2 export)

**Comment by waynebruce0x** [comment]
> i checked rvirkitty yesterday and it seems like their contract deposits the pool2 tokens into hucleberry's masterchef, so counting the pool2 tokens on their masterchef would miss some tokens (would also be great to split these into a pool2 export)

Added

File: projects/idleDao.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+
+const sdk = require('@defillama/sdk');
```
Comment (0xngmi) [inline projects/idleDao.js:2]: this only counts the tokens on polygon, whereas most idle tvl is on ethereum. the current idle adapter has already been updated to have polygon

**Review by flabble-gro — COMMENTED** [review]
flabble from Gro here. Many thanks for turning this round so quickly and looks good for the new Avalanche assets. One quick thing - the current PR has the previous Ethereum parts commented out, I assume this was just for testing the Avalanche pieces but please do uncomment this before merging :)

**Comment by waynebruce0x** [comment]
> flabble from Gro here. Many thanks for turning this round so quickly and looks good for the new Avalanche assets. One quick thing - the current PR has the previous Ethereum parts commented out, I assume this was just for testing the Avalanche pieces but please do uncomment this before merging :)

yeah so sorry, I missed that, I've uncommented it now

**Comment by flabble-gro** [comment]
Is there anything we can do to help get the gro protocol part of this PR merged - we'd love to see the Avax funds shown

**Pull request merged**